SPECS-EXTENDED/lilv/lilv.spec

```diff
@@ -0,0 +1,328 @@
+%{!?_pkgdocdir: %global _pkgdocdir %{_docdir}/%{name}-%{version}}
+%global maj 0
+%bcond_with docs
+Summary:        An LV2 Resource Description Framework Library
+Name:           lilv
+Version:        0.24.14
+Release:        4%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://drobilla.net/software/lilv
+Source0:        https://download.drobilla.net/%{name}-%{version}.tar.bz2
+BuildRequires:  doxygen
+BuildRequires:  graphviz
+BuildRequires:  sord-devel >= 0.14.0
+BuildRequires:  sratom-devel >= 0.4.4
+BuildRequires:  lv2-devel >= 1.18.0
+BuildRequires:  python3
+BuildRequires:  python3-devel
+BuildRequires:  swig
+BuildRequires:  serd-devel >= 0.30.0
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  libsndfile-devel >= 1.0.0
+%if %{with docs}
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_lv2_theme
+%endif
+Requires:       lv2 >= 1.18.0
+%if %{with_check}
+BuildRequires:  lcov
+%endif
+
+# To try and deal with multilib issues from the -libs split:
+# https://bugzilla.redhat.com/show_bug.cgi?id=2052588
+Obsoletes:      lilv < 0.24.12-2
+
+%description
+%{name} is a library to make the use of LV2 plugins as simple as possible
+for applications. Lilv is the successor to SLV2, rewritten to be significantly
+faster and have minimal dependencies.
+
+%package        libs
+Summary:        Libraries for %{name}
+Obsoletes:      lilv < 0.24.12-2
```

Comment:
Not sure if this should obsolete lilv-libs? Thanks

Reply:
Removed, it is not needed.